pandas/core/indexes/multi.py

```diff
@@ -2435,7 +2453,7 @@ def _get_partial_string_timestamp_match_key(self, key):
         if isinstance(key, str) and self.levels[0]._supports_partial_string_indexing:
             # Convert key '2016-01-01' to
             # ('2016-01-01'[, slice(None, None, None)]+)
-            key = tuple([key] + [slice(None)] * (len(self.levels) - 1))
+            key = tuple((key, *([slice(None)] * (len(self.levels) - 1))))
```

Comment:
i find this less clear than the status quo.  what about `(key,) + (slice(None),) * num`?

Reply:
Yea I agree here; if there's a way to do this without unpacking would read a lot more clearly

Alternately can be split into separate lines

Reply:
updated.